### ISSUE TITLE
Add writeln command if possible

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#270]
```
It would be great if we could use GTalkSMS input method with a kind of writeln 
command which would work the same as existing write but also click the button 
adjoined to the text field, if any, like for example Send or Search, 
immediately after the text is entered into the field.
```

Original issue reported on code.google.com by `milos.ra...@gmail.com` on 10 May 2012 at 1:30

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 10 May 2012 at 8:21
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
- Adding "wappend" and "wsend" commands with associated help.
        wappend : add text to the current field
        wsend : set the text and validate the field. Only validate if the text is empty.

- Change "\n" by a cariage return in the field.
```

Original comment by `Florent....@gmail.com` on 10 May 2012 at 8:46
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
This command is included in the new beta version (see download section) if you 
don't want the wait for the next release.
```

Original comment by `Florent....@gmail.com` on 21 May 2012 at 7:23

**Comment by GoogleCodeExporter** [comment]
```
Fixed with GTalkSMS v4.0 and released on market
```

Original comment by `Florent....@gmail.com` on 28 Jul 2012 at 4:20
- Changed state: **Deployed**

**Comment by GoogleCodeExporter** [comment]
```
Fixed with GTalkSMS v4.0 and released on market
```

Original comment by `Florent....@gmail.com` on 28 Jul 2012 at 4:20
